### PR TITLE
[V2] Load smoke tests from data file

### PR DESCRIPTION
**Pull request by aemous**
*Description of changes:*
Replaced the hard-coded smoke tests with a JSON data file.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/integration/test_smoke.py [hunk 1]
```diff
@@ -12,109 +12,20 @@
...
 import os
-import random
 import time
+import json
 
 import pytest
 
 from awscli.testutils import aws
 
+INTEG_TESTS_DIR = os.path.dirname(os.path.join(os.path.abspath(__file__)))
```
Comment (hssyoo) [inline tests/integration/test_smoke.py:22]: It looks like we don't need `os.path.join` here.

File: tests/integration/smoke-test.json [hunk 2]
```diff
@@ -0,0 +1,332 @@
+{
```
Comment (hssyoo) [inline tests/integration/smoke-test.json:1]: nit: Let's name this file `smoke-tests.json`

**Review by hssyoo — CHANGES_REQUESTED** [review]
Looks solid, just had a couple minor comments.

File: tests/integration/smoke-test.json (hunk 2 above)
Comment (aemous) [reply tests/integration/smoke-test.json]: I've updated this in my latest revision.

File: tests/integration/test_smoke.py (hunk 1 above)
Comment (aemous) [reply tests/integration/test_smoke.py]: Updated in latest revision

File: tests/integration/test_smoke.py [hunk 3]
```diff
@@ -12,109 +12,20 @@
...
-]
-
-
-# These services require a particular region to run.
-REGION_OVERRIDES = {
-    'route53domains': 'us-east-1'
-}
+with open(os.path.join(INTEG_TESTS_DIR, "smoke-test.json"), "r") as definitions_file:
+    SMOKE_TESTS = json.load(definitions_file)
+    COMMANDS = [" ".join(SMOKE_TESTS["Commands"][i]) for i in range(len(SMOKE_TESTS["Commands"]))]
```
[the remark this answers is not in the document]
Comment (aemous) [reply tests/integration/test_smoke.py]: Updated in latest revision

**Review by hssyoo — APPROVED** [review]
🏆

**Pull request merged**